src/VirtoCommerce.Platform.Web/Controllers/Api/ChangeLogController.cs

```diff
@@ -51,12 +52,39 @@ public ActionResult<LastModifiedResponse> GetLastModifiedDate([FromQuery] string
             var result = new LastModifiedResponse
             {
                 Scope = scope,
-                LastModifiedDate = _lastModifiedDateTime.LastModified.UtcDateTime
+                LastModifiedDate = _lastModifiedDateTime.GetLastModified().UtcDateTime
             };
 
             return Ok(result);
         }
 
+        [HttpPost]
+        [Route("~/api/changes/changedentities")]
+        [AllowAnonymous]
+        public ActionResult<ChangedEntitiesResponse> ChangedEntities(ChangedEntitiesRequest changedEntitiesRequest)
+        {
+            var result = new ChangedEntitiesResponse()
+            {
+                EntitiesResponses = changedEntitiesRequest.EntitiesNames.Select(x => new LastModifiedEntityResponse { EntityName = x, LastModifiedDate = _lastModifiedDateTime.GetLastModified(x).UtcDateTime })
+                    .Where(x => x.LastModifiedDate >= changedEntitiesRequest.After)
```

Comment:
One more thing: to get all changed entities since some date, storefront will need to pass all known entity names with each request. Would be nice to treat empty entity names list as all of known entities to reduce serialization cost and network traffic (though current implementation probably won't allow this, because there is no registry with all known entities).